AndroidStudioProjects/BMIApp/app/src/main/java/com/example/bmiapp/ui/input/InputFragment.kt

```diff
@@ -28,26 +33,33 @@ class InputFragment : Fragment() {
     //bmi
     private var bmi: String = ""
     //sharedのkeyリスト
-    private lateinit var keyList: MutableList<String>
+    private var keyList: MutableList<String>? = null
```

Comment:
```suggestion
    private val keyList: MutableList<String> = mutableListOf()
```
こうすると、null考慮不要になりますよー